seata-spring-boot-starter/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -150,6 +150,24 @@
       "sourceType": "io.seata.spring.boot.autoconfigure.properties.client.UndoProperties",
       "defaultValue": true
     },
+    {
+      "name": "seata.client.undo.compress.enable",
+      "type": "java.lang.Boolean",
+      "sourceType": "io.seata.spring.boot.autoconfigure.properties.client.UndoCompressProperties",
+      "defaultValue": true
+    },
+    {
+      "name": "seata.client.undo.compress.type",
```

Comment:
Please add optional values for this configuration in `hints` below. like `seata.transport.compressor`

Reply:
done